src/vs/workbench/contrib/files/browser/views/explorerView.ts

```diff
@@ -335,6 +343,13 @@ export class ExplorerView extends ViewletPanel {
 
 		this._register(this.tree.onContextMenu(e => this.onContextMenu(e)));
 
+		this._register(this.tree.onDidScroll(e => {
+			let editable = this.explorerService.getEditable();
```

Comment:
Why not use `getEditableData`?
That API is already there and gives you everything you need.

Reply:
`getEditableData` just return the data, I need the stat so I can call `this.tree.getRelativeTop(editable.stat)`